Python/Batch/sample4_job_scheduler.cfg

```diff
@@ -0,0 +1,6 @@
+[DEFAULT]
+shoulddeletejob=false
+poolvmsize=STANDARD_D1_V2
+poolvmcount=1
+shoulddeletecontainer=false
+shoulddeletepool=false
```

Comment:
Since you use autopool, `shoulddeletepool` is sorta the same as `shoulddeeltejob`, right? For example I am not sure you use the boolean derived from this at all. Maybe just get rid of it?

Reply:
yes - got rid of shoulddeletepool